ops/terraform/services/server/insights/api-requests/glue.tf

```diff
@@ -1158,17 +1158,17 @@ module "glue-table-api-requests" {
       "comment" = ""
     },
     {
-      "name"    = "mdc_jpa_query_bene_count_by_year_month_part_d_contract_id_duration_milliseconds",
+      "name"    = "mdc_jpa_query_bene_exists_by_year_month_part_d_contract_id_duration_milliseconds",
       "type"    = "string",
       "comment" = ""
     },
     {
-      "name"    = "mdc_jpa_query_bene_count_by_year_month_part_d_contract_id_duration_nanoseconds",
+      "name"    = "mdc_jpa_query_bene_exists_by_year_month_part_d_contract_id_duration_nanoseconds",
       "type"    = "string",
       "comment" = ""
     },
     {
-      "name"    = "mdc_jpa_query_bene_count_by_year_month_part_d_contract_id_record_count",
+      "name"    = "mdc_jpa_query_bene_exists_by_year_month_part_d_contract_id_record_count",
```

Comment:
I'll have to meditate on this further, but we might want to reconsider what merely _appears_ to be a rename. I believe this is somewhat destructive in nature, e.g. a deletion followed by a creation. Granted, it's merely `test`, but I can't query the data that should otherwise be available because the glue table doesn't include the old count fields.

That said, the solutions looks **very** promising with the limited data you've produced so far!

Reply:
I feel as though it may be a bandaid to retain these fields for historic reasons (and I think you already know this), so is the solution to add these back in and address that concern in a different task? or do you think it's not too problematic to keep these fields in the glue.tf when they don't appear elsewhere in the codebase?

Reply:
As discussed, we elected to keep the old fields and expand the comment field. Both fields are available–this looks great!